tests/test_operations.py

```diff
@@ -267,5 +272,67 @@ def _put_gs_file(self, key, data, content_type="blah"):
         with io.BytesIO(data) as fh:
             gs_blob.upload_from_file(fh, content_type="application/octet-stream")
 
+
+@testmode.integration
+class test_operations_integration(TestBundleApi, TestAuthMixin, DSSAssertMixin, DSSUploadMixin):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.app = ThreadedLocalServer()
+        cls.app.start()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.app.shutdown()
+
+    def setUp(self):
+        Config.set_config(BucketConfig.TEST)
+        self.s3_test_fixtures_bucket = get_env("DSS_S3_BUCKET_TEST_FIXTURES")
+        self.gs_test_fixtures_bucket = get_env("DSS_GS_BUCKET_TEST_FIXTURES")
+
+    def test_checkout_operations(self):
```

Comment:
A preferable testing strategy would be to mock `bundle_checkout` and `file_checkout`, which are already tested elsewhere. This would avoid the need to upload test bundles, reduce redundancy, and the tests would run faster.